spec/features/view_peeps_spec.rb

```diff
@@ -0,0 +1,23 @@
+feature "View peeps" do
+  scenario "A maker can view the peeps in reverse cronological order" do
+    sign_up
+    sign_in
+    post_peep
+
+    visit "/peeps/new"
+    fill_in "content", with: "Second peep"
+    click_button "Post"
+
+    expect(current_path).to eq "/peeps"
+    expect("Second peep").to appear_before("First peep")
```

Comment:
Like the use of the 'orderly' gem to make the test more readable.
There is a way to test using CSS' nth child syntax without the use of a gem.